motoko/canister_logs/Makefile

```diff
@@ -20,9 +20,35 @@ upgrade: build
 .PHONY: test
 .SILENT: test
 test: install
-	dfx canister call CanisterLogs print 'hi'
+	# Test print via update call.
+	dfx canister call CanisterLogs print 'print via update'
 	dfx canister logs CanisterLogs \
-		| grep 'hi' && echo 'PASS'
+		| grep 'print via update' && echo 'PASS'
+
+	# Test print via replicated query call.
+	dfx canister call --update CanisterLogs print_query 'print via replicated query'
+	dfx canister logs CanisterLogs \
+		| grep 'print via replicated query' && echo 'PASS'
+
+	# Test print via non-replicated query call should NOT record the message.
+	dfx canister call --query CanisterLogs print_query 'print via non-replicated query'
+	! dfx canister logs CanisterLogs \
+		| grep 'print via non-replicated query' && echo 'PASS'
+
+	# Test trap, ignore failed call output.
```

Comment:
> ignore failed call output

what does this comment mean?

Reply:
I have update those comments with `-dfx`, PTAL.

Reply:
Could you please replace "failed call output" by "failed dfx command output" if this is what you mean?

Reply:
Done here: https://github.com/dfinity/examples/pull/929/files#diff-2dc8abd0614871d2c2ed078f6d1aff60e18d5ee57e292ea61b79b007316be7e7R39